### PR TITLE
Workflow Automated Test - LRQA-68843 | master

### PR DESCRIPTION
**Pull request by evanilson-santana**

### CODE REVIEW

**Comment by evanilson-santana** [comment]
ci:test:sf

**Comment by evanilson-santana** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 1]
```diff
@@ -252,12 +252,40 @@ definition {
...
+			userEmailAddress = "usersn@liferay.com");
+
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
 
-		// TODO LRQA-68843 CanAssignTaskToOtherUser pending implementation
+		Workflow.assignByDetails(user = "usersn (userfn userln)");
 
+		Alert.viewSuccessMessageText(successMessage = "Your request completed successfully.");
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:288]: The step 7 can be extended, with this you are checking the success message, but it's not making a assert if at the new user, there is a blog assigned to him. You can logout and log in to check if the blogs is present there.

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Review by lucasperj — APPROVED** [review]

**Comment by lucasperj** [comment]
Just started reviewing :)

**Comment by lucasperj** [comment]
Combined and sent on [PR#3](https://github.com/lucasperj/liferay-portal/pull/3). Thank you!

**Pull request closed without merging**